docs/snapshotting/snapshot-support.md

```diff
@@ -156,7 +156,28 @@ Now that the microVM is paused, you can create a snapshot, which can be either a
 one or a `diff` one. Full snapshots always create a complete, resume-able snapshot of
 the current microVM state and memory. Diff snapshots save the current microVM state
 and the memory dirtied since the last snapshot (full or diff). Diff snapshots are not
-resume-able, but can be merged into a full snapshot using external (provided) tooling.
+resume-able, but can be merged into a full snapshot. In this context, we will refer to
+the base as the first memory file created by a `/snapshot/create` API call and the
```

Comment:
Maybe it will be useful to specify here that `the base` can be either the first mem file created by a diff snapshot, either a mem file created by a full snapshot.

Reply:
I thought about this, but they are both "the first memory file created by a `/snapshot/create` API call" and it sounded too complicated and confusing. I found this description to be simpler while also being correct. Let me know what you think @sandreim  @dianpopa  @luminitavoicu.

Reply:
That makes sense. As far as I am concerned, we can merge this with the current state.